### PR TITLE
Refactor.space.wizard.move.location

### PR DESCRIPTION
**Pull request by corinnekrych**
Fixes https://github.com/fabric8-ui/fabric8-ui/issues/1838

### CODE REVIEW

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-1840-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1840-fabric8-ui.badger.fabric8.io

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-1840-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1840-fabric8-ui.badger.fabric8.io

File: src/app/app.module.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ import { realmProvider }                 from './shared/realm-token.provider';
 import { ConfigStore }               from './base/config.store';
 import { ErrorService }              from './layout/error/error.service';
 import { ProfileService }            from './profile/profile.service';
-import { SpaceWizardModule }         from './space-wizard/space-wizard.module';
+import { SpaceWizardModule }         from './space/wizard/space-wizard.module';
```
Comment (joshuawilson) [inline src/app/app.module.ts]: Do we still need the module named "space-wizard"? I'm good with it, just wanted to confirm.

File: src/app/space/wizard/services/fabric8-app-generator.client.mock.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { IAppGeneratorResponse } from "app/space-wizard/models/contracts/app-generator-response";
+import { IAppGeneratorResponse } from "..//models/contracts/app-generator-response";
```
Comment (joshuawilson) [inline src/app/space/wizard/services/fabric8-app-generator.client.mock.ts]: should this have 2 slashes?

Comment (corinnekrych) [inline src/app/space/wizard/services/fabric8-app-generator.client.mock.ts]: good point. this import is invalid and not used. let me clean unused import in wizard.

File: src/app/app.module.ts (hunk 1 above)
Comment (corinnekrych) [inline src/app/app.module.ts]: Let's leave it for now. It might help to differentiate with patternfly wizard (once used).

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-1840-5 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1840-fabric8-ui.badger.fabric8.io

**Pull request merged**